js/Track/library/File/BIGWIG.js

```diff
@@ -0,0 +1,5 @@
+Genoverse.Track.File.BIGWIG = Genoverse.Track.Graph.Bar.extend({
+  model  : Genoverse.Track.Model.File.BIGWIG,
+  name   : 'bigwig',
+  height : 100
+})
```

Comment:
I have seen files with both so adding this line will then allow suppor for both extensions